commands/command_fetch.go

```diff
@@ -28,12 +28,17 @@ func fetchCommand(cmd *cobra.Command, args []string) {
 
 	if len(args) > 0 {
 		// Remote is first arg
+		if err := git.ValidateRemote(args[0]); err != nil {
+			Panic(err, fmt.Sprintf("Invalid remote name '%v'", args[0]))
+		}
```

Comment:
Instead of `Panic()`, I think `Exit()` would be more appropriate. We don't need to show a stack trace or anything like that, do we?

Also, you can use `%q` in the format string to have the value quoted: `"Invalid remote name %q"`